mobile-v3/android/fastlane/Fastfile

```
@@ -0,0 +1,43 @@
+  desc "Deploy to new version to Firebase App Distribution"
+  lane :app_distribution do
+    update_fastlane
+    sh "bundle exec fastlane update_plugins"
+    sh "flutter build appbundle --flavor airqo" 
+    firebase_app_distribution(
+      app: ENV['MOBILE_ANDROID_ID_FIREBASE'],
+      android_artifact_path:"../build/app/outputs/bundle/airqoRelease/app-airqo-release.aab",
+      release_notes_file: "fastlane/release_notes.txt",
+      service_credentials_file: "fastlane/firebase-app-distribution.json",
+      android_artifact_type: "AAB",
+      groups: "airqo-team",
```

rubocop (1.69.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)